components/map/overview.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Overview
+page_title: Map Overview
+description: Discover the Blazor Map and explore the examples.
+slug: components/map/overview
+tags: telerik,blazor,map,overview
+published: True
+position: 0
+---
+
+# Map Overview
+
+The Map displays geospatial information organized in layers.
+
+The component provides tile layers, shape (vector) layers, bubble layers, and marker layers.
+
+## Creating Blazor Map
+
+1. Use the `TelerikMap` tag to add the component to your razor page.
+
+2. Add the `MapLayer` tag nested inside `MapLayers`.
+
+3. Set the `Type` property.
+
+4. Set the `Attribution` and `Subdomains` properties.
+
+5. Provide the `UrlTemplate` property.
```

Comment:
Consider providing some information on the format of the UrlTemplate. I think it's something custom in our Map component and should be documented somewhere. 

Take inspiration from the Format Labels in the Chart docs.

Reply:
It is documented in the Layers -> Overview -> Parameters section. I will at least link the section to the property.